web/src/users/add_url.rs

```diff
@@ -0,0 +1,32 @@
+use axum::{
+    extract::{Form, State},
+    response::{IntoResponse, Redirect},
+};
+use db::{urls::add_url, users::User};
+use serde::Deserialize;
+
+use crate::AppState;
+
+#[derive(Deserialize)]
+pub struct ArticleForm {
+    content: String,
```

Comment:
Can we rename this field to be `url` or something? And also change the form label and stuff?

At first I wasn't sure if the form wanted me to like copy paste the article body and I was a bit confused 😆